errors/src/test/java/com/palantir/conjure/java/api/errors/QosExceptionTest.java

```diff
@@ -47,4 +48,42 @@ public Class visit(QosException.Unavailable exception) {
                 .isEqualTo(QosException.RetryOther.class);
         assertThat(QosException.unavailable().accept(visitor)).isEqualTo(QosException.Unavailable.class);
     }
+
+    @Test
+    public void testReason() {
+        QosReason reason = new QosReason("reason");
+        try {
+            throw QosException.throttle(reason);
+        } catch (QosException.Throttle e) {
+            assertThat(e.getReason().getName()).isEqualTo(reason.getName());
+        }
```

Comment:
We don't need to throw here:
```suggestion
        assertThat(QosException.throttle(reason).getReason()).isEqualTo(reason);
```

Reply:
Nice. Updated.